discourse_migration/userguide.md

```diff
@@ -45,48 +45,49 @@ are the typical use cases:
 
 This table explains the mapping from mailing lists to categories in Discourse.
 The email addresses of these categories will remain the same, after the
+migration.  Deprecated lists will become read-only as part of the Discourse
 migration.
 
 | Mailing lists         | category in Discourse |
 |-----------------------|--------------------|
 | All-commits           | no migration at the moment |
 | Bugs-admin            | no migration at the moment |
 | cfe-commits           | no migration at the moment |
-| cfe-dev               | llvm-project/clang |
-| cfe-users             | **TODO** llvm-project/clang or create new category? |
-| clangd-dev            | llvm-project/clangd |
-| devmtg-organizers     | **TODO** create new category |
+| cfe-dev               | Clang Compiler & Tools |
+| cfe-users             | Clang Compiler & Tools/Using Clang |
+| clangd-dev            | Clang Compiler & Tools/clangd |
+| devmtg-organizers     | Deprecated |
 | Docs                  | infrastructure/documentation |
-| eurollvm-organizers   | **TODO** create new category |
+| eurollvm-organizers   | Deprecated |
 | flang-commits         | no migration at the moment |
-| flang-dev             | llvm-project/flang |
+| flang-dev             | LLVM Based Subprojects/flang |
```

Comment:
I'd recommend naming this just "Subprojects / Flang".  No need for "LLVM Based".  Similarly, I think we should rename "LLVM Incubator" to just "Incubator".  The extra verbosity shows up in email subject lines, like this:

`Subject: [LLVM] [LLVM Incubator/CIRCT] Proposal for Per-instance Metadata`

It would be better to have:

`Subject: [LLVM] [Incubator/CIRCT] Proposal for Per-instance Metadata`
`Subject: [LLVM] [Subproject/Flang] Fancy fortran topic`

I'd also recommend capitalizing "Flang" in the topic name just like we do "Clang" for consistency.  It is talking about the subproject, not the executable.

Reply:
I do think we should consider if most people are really using email though when deciding how to name things. This particular case seems fine though.